recipes/conda-bash-completion/Makefile

```diff
@@ -0,0 +1,18 @@
+# Aliases for maintaining this conda package
```

Comment:
Please remove this file if it is not being used in the build. The conda-forge bots will do all of the maintenance.